### ISSUE TITLE
Issues with Training and Claimed Accuracy

### ISSUE THREAD

**Issue by MalayAgarwal-Lee** [DariusAf/MesoNet#27]
Hi, @DariusAf,

I'm trying to replicate the results of this model but can't seem to get the accuracy to be anywhere near the numbers claimed in the paper. I'm using the dataset provided in the readme.

I believe that my training code is correct and there's nothing wrong with it. The only "novel" thing in the code is a custom implementation of a learning rate schedule which decreases the learning rate by an order of magnitude every 1000 iterations, as stated in the paper.

Still, my validation accuracy is stuck at somewhere around ~75% and doesn't budge from that point.

Can you help me, please?

**Comment by MalayAgarwal-Lee** [comment]
Hi, @DariusAf 

I'd really appreciate it if you could take a look at this. A big portion of my final year grade depends on this model succeeding.

Thanks.

**Comment by MalayAgarwal-Lee** [comment]
Never mind, the model is working great now!

**Comment by DariusAf** [comment]
Well, my take would be that research is not about having a model succeeding, and thus getting a good grade (or equivalently nowadays, beating the SoTA). Finding why it potentially does not work is also a worthy contribution.
Thus, what was wrong?
